### PR TITLE
Update design of `CancellationForm`

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Update design of `CancellationForm` to match Figma.

Add updated inputs matching Figma.

## Justify why they are needed

So we can show that some parts of the UI is ready!

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-1822]: https://hedvig.atlassian.net/browse/GRW-1822?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #972** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/972" target="_blank">Graphite</a> 
    * **PR #973** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/973" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/973?utm_source=stack-comment).

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/DateInput.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          <StyledLabel htmlFor={identifier}>{label}</StyledLabel>
+          <StyledInput type="date" id={identifier} {...inputProps} />
+        </CheckboxHeader>
+        {children}
+      </Space>
+    </InputWrapper>
+  )
+}
+
+const InputWrapper = styled.div(({ theme }) => ({
```
Comment (alebedev) [inline apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/DateInput.tsx:25]: Nit: we can reduce DOM size if we make it `styled(Space)`

**Review by alebedev — APPROVED** [review]
LGTM.  One nit comment

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/973?utm_source=stack-comment).


Job Reference: tnRk3yNxxdFZ5CMl5L4E

**Pull request merged**